Packs/FireEyeNX/IncidentTypes/incidenttype-FireEye_NX_Incident.json

```diff
@@ -0,0 +1,26 @@
+{
+  "id": "FireEye NX Incident",
```

Comment:
Please remove the word 'incident' from the id+name+file name, this will cause the build to fail since this terminology can be changed in the server based on customers terminology.

Reply:
sure @mayagoldb